src/qae/object-properties.js

```diff
@@ -78,25 +72,25 @@ const properties = {
   },
   /**
    * Holds general styling
-   * @type {?Styling[]}
```

Comment:
I think Component is better, so I choose to match Component with    the declaration file